lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,186 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
+
+Function CheckModified-File($url, $dest, $headers, $credentials, $timeout, $use_proxy, $proxy) {
```

Comment:
The reason I wrote this patch is because that is the behaviour on Windows *and* Unix. And we explicitly did not want the proxy to be used. What I still need to do is the same for win_uri and uri. (use_ssl appears to magically work, but is undocumented)